routers/web/user/auth.go

```diff
@@ -90,14 +90,23 @@ func AutoSignIn(ctx *context.Context) (bool, error) {
 
 	isSucceed = true
 
+	newSess, _ := ctx.Session.RegenerateID(ctx.Resp, ctx.Req)
+	oldID := ctx.Session.Get("_old_uid")
```

Comment:
1. should it be `oldUID`?
2. I haven't understood the real usage of `_old_uid` ....

Reply:
I can't completely remember it's uses but I've changed the code to avoid having to copy this.